### ISSUE TITLE
StillMaintained-style badges

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [badges/shields#607]
So, it's pretty simple to create a static ‘this is unmaintained’ badge:

![](https://img.shields.io/badge/maintained%3F-no!-red.svg?style=flat): `https://img.shields.io/badge/maintained%3F-no!-red.svg?style=flat`

However, what I'd *really* like to see, as a hopefully-easy-to-implement feature, is ‘yes this is maintained’ badges that *expire*:

https://img.shields.io/maintenance/STATUS/2015.svg

If `STATUS` is `yes`, then during 2015, this shows ![](https://img.shields.io/badge/maintained%3F-yes-brightgreen.svg?style=flat); but let's say three months into 2016, any badges with a number lower than `2015.svg` show ![](https://img.shields.io/badge/maintained%3F-stale_(as_of_2016)-lightgrey.svg?style=flat) … and then perhaps three months into 2017, `2015.svg` will resolve to ![](https://img.shields.io/badge/maintained%3F-no!-red.svg?style=flat).

Basically, I want a way to actively continue to signal that a project is maintained, that is *trustworthy* to the end-user: at the moment, I'm including the year in the manual badge I'm adding to my projects; but this is bulky and ugly, IMHO. It'd be much better for ‘yes!’ to simply mean yes, by definition, and to become ‘no!’ automatically.

> Alternatively, the ‘value’ of the badge could simply be the year; and the only thing that changes could be the colour; but, personally, I like the ability to have custom words: `https://img.shields.io/maintenance/feature-frozen/2015.svg`, for instance.

**Comment by espadrine** [comment]
That is smart.

I implemented it and it is now live. Your link is now functional, although this one may be of better use:

https://img.shields.io/maintenance/yes/2016.svg

**Comment by ELLIOTTCABLE** [comment]
Damn, that was fast!

@espadrine, you rock. Keep up the slick work.
